src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Button, Nav, Navbar } from 'react-bootstrap';
+
+import { Profile } from '../App';
+import LoginButton from '../LoginButton';
+import styles from './styles.module.scss';
+
+type Props = {
+  onLogOut: () => void;
+  profile: null | Profile;
+};
+
+export class NavbarBase extends React.Component<Props> {
+  render() {
+    const { onLogOut, profile } = this.props;
+
+    return (
+      <Navbar bg="dark" className={styles.Navbar} expand="lg" variant="dark">
+        <Navbar.Brand href="#">addons-code-manager</Navbar.Brand>
+        <Nav>
+          {profile ? (
+            <Nav.Link className="Navbar-username">{profile.name}</Nav.Link>
```

Comment:
I made it a `Nav.Link` because that's what works out of the box with Bootstrap to make it appear the way we want. But it doesn't need to be a link so I've changed it to use `Navbar.Text` and some custom styling.